pipes/WDL/dx-launcher/demux_launcher.yml

```diff
@@ -19,6 +19,10 @@ inputSpec:
   class: string
   default: /
   help: Output folder in the root project. For this applet, the --destination should be set to / and this input should be used to control the output folder.
+- name: sequencing_center
+  class: string
+  optional: true
+  help: Optional- the name of the sequencing center specified in the demultiplexing output (bam files)
```

Comment:
Note in the help string here that the value should avoid any spaces or special characters? Beats having to escape and support all those scenarios.

Reply:
We could process the value passed for sequencing center via `util.file.string_to_file_name()` in illumina.py?

Reply:
I'm not assuming that this yml has access to our python `util.file`. In fact the runSpec looks a bit more bash-like. I'm not worried about the downstream stuff, I'm more worried about your actual `runcmd` just below if someone entered a sequencing center like "My Sequencing Center".

Reply:
That makes sense. I suppose I'll escape the input in illumina.py just to be safe, and also mention in the help text to avoid spaces and special characters.